scripts/globals/abilities/reward.lua

```diff
@@ -115,14 +115,14 @@ abilityObject.onUseAbility = function(player, target, ability, action)
             pet:delStatusEffect(xi.effect.SILENCE)
         end,
 
-        [15095] = function (x) -- monster jackcoat
+        [15095] = function() -- monster jackcoat
             -- This will remove Weight, Slow and Silence from the pet.
             pet:delStatusEffect(xi.effect.WEIGHT)
             pet:delStatusEffect(xi.effect.SLOW)
             pet:delStatusEffect(xi.effect.SILENCE)
         end,
 
-        [14508] = function (x) -- monster jackcoat +1
+        [14508] = function() -- monster jackcoat +1
```

Comment:
I always wondered why the x was even there

Reply:
Assume that future developers will always use the ugliest code first as a template for additional work.